apis/opts/labels.go

```diff
@@ -21,8 +20,10 @@ func ParseLabels(labels []string) (map[string]string, error) {
 
 func parseLabel(label string) ([]string, error) {
 	fields := strings.SplitN(label, "=", 2)
-	if len(fields) != 2 {
-		return nil, fmt.Errorf("invalid label %s: label must be in format of key=value", label)
+	// Only input key without value
+	if len(fields) == 1 {
+		fields = append(fields, "")
+		return fields, nil
```

Comment:
duplicate return here. just use global return 

BTW, do we need error here?

Reply:
I have checked the `env` command and it has solved the empty value problem.

Do you mean I should modify the code logic of `label` command as same as `env`  command?

Reserve the parameter of error is beacause I don't know whether it is essential for future development.

I will modify the return statement. Thanks.